literal/utils.go

```diff
@@ -14,8 +17,8 @@ func NewBool(value bool) (expr.Literal, error) {
 	return expr.NewPrimitiveLiteral[bool](value, false), nil
 }
 
-func NewInt8(value int8) (expr.Literal, error) {
-	return expr.NewPrimitiveLiteral[int8](value, false), nil
+func NewInt8(value int8) expr.Literal {
+	return expr.NewPrimitiveLiteral[int8](value, false)
 }
 
 func NewInt16(value int16) (expr.Literal, error) {
```

Comment:
If the other signatures are changing, should these change too?

Reply:
Yes, Fixed them.